apps/ledger-live-mobile/fastlane/Fastfile

```diff
@@ -187,10 +187,11 @@ platform :ios do
 
       if (options[:ci])
         sh(
-          <<-SHELL
-           echo "IOS_BUILD_NUMBER=#{build_number+1}" >> "$GITHUB_ENV"
-          SHELL
-        )
+            <<-SHELL
+              new_build_number="IOS_BUILD_NUMBER=#{build_number+1}"
```

Comment:
I also don't understand why this fixes the issue - are you able to explain?

Reply:
Yes sure ! so , the main problem is that the redirection to `>> env:$GITHUB_ENV` is  not injecting the build number in the right environment variable that will be used in the next steps for slack message notification. 
Previously the redirection was directly to `>> $GITHUB_ENV` , the problem with this is at fastlane runtime, it encounters an `Ambiguous Redirect error` which is the case of this [ticket](https://ledgerhq.atlassian.net/browse/LIVE-14473) . 
An official solution was to use the $GITHUB_ENV between `"$GITHUB_ENV"` like in the documentation of setting environment variables. The problem with that, is on fastlane runtime, it occures a `No such directory ...` issue.
So to workaround it, the solution (and the only one that i found related to this specific issue ) it to pass by a temporary variable, set the new build number and use the `tee` to append it at the end of the $GITHUB_ENV path the new build number which is a workable solution. Same issue in this [discussion](https://github.com/orgs/community/discussions/36746)

Reply:
Okay thanks!